scriptworker/cot/verify.py

```diff
@@ -540,7 +551,7 @@ def find_task_dependencies(task, name, task_id):
         CoTError: on failure.
     """
     link = chain.get_link(task_id)
-    log.debug("Verifying {} is in {} cot artifacts...".format(path, task_id))
+    log.debug(" Verifying {} is in {} cot artifacts...".format(path, task_id))
```

Comment:
Bikeshedding: If I understand correctly, a space describes a debug type of log. Is that correct? If so, I wonder if we couldn't write a Formatter that adds it for us.